<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5MFTask</class>
 <widget class="QWidget" name="CE5MFTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main info</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="C5LineEditWithSelector" name="leState">
         <property name="Field" stdset="0">
          <string>f_state</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="C5LineEditWithSelector" name="lineEdit_9">
         <property name="Field" stdset="0">
          <string>f_qty</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="C5LineEditWithSelector" name="leProductName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="FieldRef" stdset="0">
          <string>f_productname</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="C5LineEditWithSelector" name="lineEdit_11">
         <property name="Field" stdset="0">
          <string>f_out</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="C5LineEditWithSelector" name="leStage">
         <property name="Field" stdset="0">
          <string>f_stage</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Out</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="C5LineEditWithSelector" name="leCode">
         <property name="Field" stdset="0">
          <string>f_id</string>
         </property>
         <property name="Primary" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Stage</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Date create</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Product</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="C5LineEditWithSelector" name="leStateName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="FieldRef" stdset="0">
          <string>f_statename</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="C5LineEditWithSelector" name="leStageName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="FieldRef" stdset="0">
          <string>f_stagename</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="C5LineEditWithSelector" name="lineEdit_10">
         <property name="Field" stdset="0">
          <string>f_ready</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QTimeEdit" name="teCreate">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
         <property name="Field" stdset="0">
          <string>f_timecreate</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="C5LineEditWithSelector" name="leProduct">
         <property name="Field" stdset="0">
          <string>f_product</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Workshop</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Ready</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="C5LineEditWithSelector" name="leWorkshop">
         <property name="Field" stdset="0">
          <string>f_workshop</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="C5DateEdit" name="leDateCreate">
         <property name="Field" stdset="0">
          <string>f_datecreate</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Responsible</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="C5LineEditWithSelector" name="leResponsible">
         <property name="Field" stdset="0">
          <string>f_responsible</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="C5LineEditWithSelector" name="leWorkshowName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="FieldRef" stdset="0">
          <string>f_workshopname</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="C5LineEditWithSelector" name="leResponsibleName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="FieldRef" stdset="0">
          <string>f_workshopname</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="C5TableWidget" name="tbl">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <column>
          <property name="text">
           <string>Key</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leCode</tabstop>
  <tabstop>leState</tabstop>
  <tabstop>leDateCreate</tabstop>
  <tabstop>leWorkshop</tabstop>
  <tabstop>leStage</tabstop>
  <tabstop>leProduct</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>tbl</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>teCreate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
